src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-list/rgw-user-list.component.html

```diff
@@ -51,6 +51,6 @@
                [align]="'top'"
                i18n-description
                i18n>
-    {{row.account?.name}}
+    <span>{{row.account?.name}}</span>
```

Comment:
why span ? cant we add an `id` to access ?

Reply:
it is rendering with space at start and last, that's why ` account name `

Reply:
maybe if you move this line above, the space wouldn't come?

```suggestion
    i18n>{{row.account?.name}}

Reply:
fixed